src/main/java/org/javafxports/jfxmobile/plugin/Gvm.java

```diff
@@ -83,7 +83,8 @@ public static void build(String target, Project project ) {
                     .forcelinkClasses(Arrays.asList(config.getForcelinkClasses()))
                     .jarDependencies(config.getJarDependecies())
                     .frameworks(Arrays.asList(config.getFrameworks()))
-                    .frameworksPaths(Arrays.asList(config.getFrameworksPaths()));
+                    .frameworksPaths(Arrays.asList(config.getFrameworksPaths()))
+                    .minOSVersion(config.getMinOSVersion());
```

Comment:
what happens if the user doesn't specify this? Null? Empty string? "0" ?

Reply:
If null, gluonTools will set "8" as default value. Most of the config variables are not set initially.